modules/Core/includes/endpoints/CreateWebhooksEndpoint.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * @param string $name The name of the webhook
+ * @param string $url The url of the webhook
+ * @param string $type The webhook type
+ * @param array $events A list of events the webhook should receive
+ *
+ * @return string JSON Array
+ */
+class CreateWebhooksEndpoint extends KeyAuthEndpoint {
+
+    public function __construct() {
+        $this->_route = 'webhooks/create';
+        $this->_module = 'Core';
+        $this->_description = 'Create a new webhook';
+        $this->_method = 'POST';
+    }
+
+    public function execute(Nameless2API  $api): void {
+        // Validation
+        $api->validateParams($_POST, ['name', 'url', 'type', 'events']);
```

Comment:
Does the `Validate::check(...)` following this not do the same thing?

Reply:
It does I believe. Will remove that